src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -610,6 +610,24 @@ def cardinality(self, algorithm=None, extension_degree=1):
 
     order = cardinality  # alias
 
+    @cached_method
+    def multiplication_by_p_isogeny(self):
+        r"""
+        Return the multiplication-by-\(p\) isogeny.
```

Comment:
Looking at the [documentation preview](https://deploy-preview-37096--sagemath.netlify.app/html/en/reference/arithmetic_curves/sage/schemes/elliptic_curves/ell_finite_field#sage.schemes.elliptic_curves.ell_finite_field.EllipticCurve_finite_field.multiplication_by_p_isogeny), this syntax appears to be unsupported.

```suggestion
        Return the multiplication-by-`p` isogeny.
```